### DISCUSSION TITLE
Is there a way to update find_focused file option?

### DISCUSSION THREAD

**Discussion by linrongbin16** [nvim-tree/nvim-tree.lua#1975] [Q&A] [answered]
Hi, I have read doc and know that, there's option to always focused current file, or find file (`NvimTreeFindFile`).

But that seems a one-time key, is there a way to update this option.

For example, before I set this option, the file is not focused. and nvim-tree will not always follow my current editing file.
After I set this option, nvim-tree will always follow my current editing file, no need to run `NvimTreeFindFile` any more.

Maybe I could simply call `require('nvim-tree').setup` to reload the configs?

**Comment by alex-courtis** [comment, marked as answer]
To confirm: are you enabling `:h nvim-tree.update_focused_file`?

That should be done in your one and only call to `setup`.
